python/dask_cudf/dask_cudf/io/tests/test_parquet.py

```diff
@@ -60,6 +60,17 @@ def test_roundtrip_from_dask(tmpdir):
     assert_eq(ddf[["y"]], ddf2)
 
 
+def test_roundtrip_from_dask_cudf(tmpdir):
+    tmpdir = str(tmpdir)
+    gddf = dask_cudf.from_dask_dataframe(ddf)
+    gddf.to_parquet(tmpdir)
+
+    # NOTE: Need `.compute()` to resolve correct index
+    #       name after `from_dask_dataframe`
```

Comment:
I'm not sure I understand this.  Is there something wrong with our metadata?

Reply:
I was also a bit confused by this.  It seems that `gddf = ddf.map_partitions(cudf.from_pandas)` will result in a dask_cudf dataframe with the `_meta` not having the same index name ans the original dask dataframe.  Is it reasonable for me to raise a separate issue?

Reply:
If you can find a quick resolution to that I think that that would be ideal.  If it ends up being more complicated then sure, let's wait.

Reply:
> If you can find a quick resolution to that I think that that would be ideal.

Okay - Looks like the index name is dropped in [`iloc` calls](https://github.com/dask/dask/blob/9c72876af571ef90ecad5df7c8bd123b06220305/dask/dataframe/io/io.py#L205) for cudf dataframes (and not for pandas).  I'll see if the fix is simple

Reply:
@mrocklin My idea for a simple fix doesn't seem to work - Lets address the index-name issue separately so I can focus on higher priority items in the short term.

Reply:
OK, sounds good.  Happy to merge on passed tests